internal/cli/auth/login.go

```diff
@@ -190,11 +196,11 @@ func (opts *LoginOpts) setUpProfile(ctx context.Context) error {
 	// Only make references to profile if user was asked about org or projects
 	if opts.AskedOrgsOrProjects && opts.ProjectID != "" && opts.OrgID != "" {
 		if !opts.ProjectExists(config.ProjectID()) {
-			return fmt.Errorf("you don't have access to this project ID or it doesn't exist")
+			return fmt.Errorf("%w", ErrProjectIDNotFound)
```

Comment:
```
return ErrProjectIDNotFound
```

Reply:
wrapping is good to add more information, if there is nothing to be added, returning the error is fine

Reply:
yes, much better - thanks!